xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -35,8 +35,34 @@ void CVideoLayerBridgeDRMPRIME::Disable()
   m_DRM->AddProperty(plane, "FB_ID", 0);
   m_DRM->AddProperty(plane, "CRTC_ID", 0);
 
-  // disable HDR metadata
   auto connector = m_DRM->GetConnector();
+
+  bool result;
+  uint64_t value;
+  std::tie(result, value) = connector->GetPropertyValue("Colorspace", "Default");
+  if (result)
+  {
+    CLog::Log(LOGDEBUG, "CVideoLayerBridgeDRMPRIME::{} - setting connector colorspace to Default", __FUNCTION__);
+    m_DRM->AddProperty(connector, "Colorspace", value);
+  }
+
+  std::tie(result, value) = connector->GetPropertyValue("output format", "RGB444");
```

Comment:
Is RGB444 the standard? Why not RGB420 or RGB422? Is there a `Default` maybe?

Reply:
The default is RGB444, and both RGB420 and 422 aren't supported by HDMI afaik?